<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>591</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Artificial Face Generation Model Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Source Image</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
         <item>
          <widget class="QLabel" name="source_image_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
           <item>
            <widget class="QLineEdit" name="image_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="image_open_pushButton">
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Driving Video</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1">
           <item>
            <widget class="QLineEdit" name="dv_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="gv_open_pushButton">
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="generate_pushButton">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Generated Video</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fps_label">
      <property name="text">
       <string>Idle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
